nncf/experimental/torch/fx/transformations.py

```diff
@@ -243,6 +243,63 @@ def _set_module_to_the_graph_module(
     return module_name_in_model
 
 
+def is_supported_batch_norm_for_training(node: torch.fx.Node):
+    """
+    Return True if the given node refers to an aten batch norm op QAT supports.
+    """
+    supported_ops = [
+        torch.ops.aten._native_batch_norm_legit.default,
+        torch.ops.aten.cudnn_batch_norm.default,
+        torch.ops.aten.miopen_batch_norm.default,
+    ]
+    return node.target in supported_ops
+
+
+def is_conv_node(node: torch.fx.Node):
+    """
+    Return whether the node refers to an aten conv op.
+    """
+    return node.op == "call_function" and node.target in [
+        torch.ops.aten.conv1d.default,
+        torch.ops.aten.conv2d.default,
+    ]
```

Comment:
Please use `_is_conv` function

Reply:
Done

Reply:
There are two `_is_conv_node` function in the file, please remove one of the functions

Reply:
Oh, I misunderstood the suggestion. Sorry about that.